priv/www/js/main.js

```diff
@@ -1539,3 +1539,11 @@ function select_queue_type(queuetype) {
     queue_type = queuetype.value;
     update();
 }
+
+function is_quorum(queue) {
+    return queue.hasOwnProperty('members');
```

Comment:
Is the `type` field not included?

Reply:
Good point. I probably hadn't notice it because it's inside `arguments` - `x-queue-type` and not available on its own. Fixed now.